korean/instagram/insta.md

```diff
@@ -0,0 +1,14 @@
+인스타그램에 인증샷을 올린다. 하트 숫자가 늘어나는 것을 보고 기분이 좋아졌다.
+
+이제 어떻게 할까요?
+
+[벽을 먹는다](eating-walls/eating-marshmallows.md)?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.